src/main/services/AxiosErrorAdapter.ts

```diff
@@ -0,0 +1,10 @@
+import { AxiosError } from 'axios';
+
+// Get details from an axios error
+export const axiosErrorDetails = (axiosError: AxiosError<{ error: string }>): string => {
+  let errorMessage = axiosError.message;
+  if (axiosError.response) {
+    errorMessage += `, ${axiosError.response.data?.error}`;
```

Comment:
Not to nit-pick, but if `axiosError.response.data?.error` is null, adding just ", " onto the error message seems a bit wrong

Reply:
Yeah good point, I'll just do a check for `axiosError.response.data?.error` in the if. What do you think?